Core/OfficeDevPnP.Core/Utilities/WebParts/IWebPartPostProcessor.cs

```diff
@@ -5,6 +5,6 @@ namespace OfficeDevPnP.Core.Utilities.WebParts
 {
     public interface IWebPartPostProcessor
     {
-        void Process(WebPartDefinition wpDefinition, File webPartPage);
+        void Process(WebPartDefinition wpDefinition, Web web);
```

Comment:
Hi Sergei,
Unfortunately this is a breaking change, because this interface is public and maybe someone else is implementing it in her/his own solutions. Can I ask you to provide an alternative implementation, which is not a breaking change? Thanks.